frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -597,9 +597,11 @@ end
 function ReaderBookmark:renameBookmark(item, from_highlight)
     if from_highlight then
         -- Called by ReaderHighlight:editHighlight, we need to find the bookmark
+        local pboxes = item.pboxes
         for i=1, #self.bookmarks do
             if item.datetime == self.bookmarks[i].datetime and item.page == self.bookmarks[i].page then
                 item = self.bookmarks[i]
+                item.pboxes = pboxes
```

Comment:
We need to save the pboxes from the (highlight) item, as we need them later to identify the PDF annotation belonging to this bookmark.